### PR TITLE
Issue 71: Badge placement

### PR DESCRIPTION
**Pull request by tmathern**
## Issue

Fix badge placement with stats block

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1
- After: https://issue-71--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1

### CODE REVIEW

File: blocks/tabview/tabview.js [hunk 1]
```diff
@@ -70,6 +76,10 @@ async function loadTabPanel(panel) {
     nestedSlides.querySelectorAll('.pairs-with-text').forEach((slideText) => {
       slideText.classList.add('pairs-with-text-badge');
     });
+
+    setTimeout(() => {
```
Comment (allentsliu) [inline blocks/tabview/tabview.js:80]: just curious, what's the purpose of adding setTimeout?

Comment (tmathern) [reply blocks/tabview/tabview.js:80]: (You can try it out without the timeout and see what happens :) )

Comment (allentsliu) [reply blocks/tabview/tabview.js:80]: I removed the timeout and it seems to still work well.

**Review by allentsliu — APPROVED** [review]

File: blocks/tabview/tabview.js (hunk 1 above)
Comment (tmathern) [reply blocks/tabview/tabview.js:80]: Then feel free to remove it once this is merged into your branch too!

**Pull request merged**